ts/routes/card-info/Revlog.svelte

```diff
@@ -174,7 +174,16 @@ License: GNU AGPL, version 3 or later; http://www.gnu.org/licenses/agpl.html
                 {/each}
             </div>
         </div>
-        {#if fsrsEnabled}{/if}
+        <!-- {#if fsrsEnabled}
```

Comment:
What's the plan here?

Reply:
For debugging the forgetting curve graph.